larray/core/array.py

```diff
@@ -2296,19 +2298,23 @@ def __iter__(self):
     def __contains__(self, key):
         return any(key in axis for axis in self.axes)
 
-    def as_table(self, maxlines=None, edgeitems=5, light=False, wide=True, value_name='value'):
-        """
+    def as_table(self, maxlines=-1, edgeitems=5, light=False, wide=True, value_name='value'):
+        r"""
         Generator. Returns next line of the table representing an array.
 
         Parameters
         ----------
         maxlines : int, optional
-            Maximum number of lines to show.
+            Maximum number of lines to show. If negative all lines are shown.
```

Comment:
I wouldn't bother supporting other negative numbers, so I would write:
`Defaults to -1 (all lines are shown).`